### PR TITLE
En 8137 delegation bughunt

### PR DESCRIPTION
**Pull request by BeniaminDrasovean**

### CODE REVIEW

File: vm/systemSmartContracts/delegation.go [hunk 1]
```diff
@@ -464,6 +470,12 @@ func (d *delegation) addNodes(args *vmcommon.ContractCallInput) vmcommon.ReturnC
 	if returnCode != vmcommon.Ok {
 		return returnCode
 	}
+
+	if len(args.Arguments) <= 0 {
```
Comment (sasurobert) [inline vm/systemSmartContracts/delegation.go:474]: < 2.

File: vm/systemSmartContracts/delegation.go [hunk 2]
```diff
@@ -541,6 +553,11 @@ func (d *delegation) removeNodes(args *vmcommon.ContractCallInput) vmcommon.Retu
 		return returnCode
 	}
 
+	if len(args.Arguments) <= 0 {
```
Comment (sasurobert) [inline vm/systemSmartContracts/delegation.go:556]: < 1

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: vm/systemSmartContracts/delegation.go (hunk 2 above)
Comment (SebastianMarian) [reply vm/systemSmartContracts/delegation.go:556]: == 0

**Review by SebastianMarian — DISMISSED** [review]

File: vm/systemSmartContracts/delegation.go (hunk 1 above)
Comment (BeniaminDrasovean) [reply vm/systemSmartContracts/delegation.go:474]: Right, done.

File: vm/systemSmartContracts/delegation.go (hunk 2 above)
Comment (BeniaminDrasovean) [reply vm/systemSmartContracts/delegation.go:556]: Done.

**Review by SebastianMarian — DISMISSED** [review]

**Review by sasurobert — DISMISSED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**